lib/AppInfo/Application.php

```diff
@@ -24,28 +24,27 @@
 
 namespace OCA\Richdocuments\AppInfo;
 
-use OC\EventDispatcher\SymfonyAdapter;
```

Comment:
@Raudius Might make sense to cherry-pick from scratch again, as it seems there were also some things dropped on the commits that the bot added. :/ SymfonyAdapter is still used below.

Reply:
Oh actually this is also missing on the stable24 backport, wondering why CI or my local test didn't catch that. That should throw an error.

Reply:
The SymphonyAdapter usage was removed on the original PR

https://github.com/nextcloud/richdocuments/pull/2328/files#diff-4eb824426a8cffb8b33b62b57744c69f32ea30ea755249a7a6170d9402885debL141

I think I just messed up the manual backport on this one when fixing the conflicts